modelina-website/src/components/playground/options/GeneralOptions.tsx

```diff
@@ -97,8 +97,15 @@ class GeneralOptions extends React.Component<
           </label>
         </li>
 
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Include change type mapping example :">
+            <p>
+              In code generation, a common task is to map the data types from the input model to the output code. One approach to accomplish this is by using type mapping.
+              <br/><br/>
+              'Change type mapping' refers to the process of transforming one data type to another data type. This is a common technique in programming when data types need to be converted to make them compatible with another system or process
```

Comment:
I think we are saying the same thing twice, maybe this makes more sense?
```suggestion
              This option includes a simple example type mapping, that maps integers to a custom type.
```

Reply:
done!! :tada: